misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathAssertionsSpec.kt

```diff
@@ -689,6 +709,53 @@ abstract class PathAssertionsSpec(
             expect(folder).isDirectoryFun()
         }
     }
+
+    describeFun(hasSameBinaryContentAs.name, hasSameTextualContentAs.name) {
+        val hasSameBinaryContentAsFun = hasSameBinaryContentAs.lambda
+        val hasSameTextualContentAsFun = hasSameTextualContentAs.lambda
+
+        context("has same binary content") {
+            it("${hasSameBinaryContentAs.name} - does not throw") withAndWithoutSymlink { maybeLink ->
+                val sourcePath = maybeLink.create(tempFolder.tmpDir.resolve("text1"))
+                val targetPath = maybeLink.create(tempFolder.tmpDir.resolve("text2"))
+                expect {
+                    expect(sourcePath).hasSameBinaryContentAsFun(targetPath)
+                }
```

Comment:
if you wrap something into `expect { }` then you create an `Expect<() -> Expect<Path>>` you would need to do an assert on it, like `toThrow<...>` or `notToThrow<...>`. However, we haven't used notToThrow so just remove it.
```suggestion
                 expect(sourcePath).hasSameBinaryContentAsFun(targetPath)
```